rails_app/config/settings/production.yml

```diff
@@ -5,14 +5,14 @@ graphite:
     default_get_metric_time: "-14d"
 
 parsers:
-    itaka:
+    www.itaka.pl:
```

Comment:
why this change again?

Reply:
This is the format that BasicParser takes, just making it up to date with `development.yml`